.github/workflows/shared_workflow.yml

```
@@ -111,10 +111,10 @@ jobs:
           ./occ a:e integration_openproject
           cd apps/integration_openproject
           # The following if block can be removed once Nextcloud no longer supports PHP 8.0
-          if [ "${{matrix.phpVersion}}" -eq 8 ]; then
-            make phpunitforphp8.0
+          if [ "${{ matrix.phpVersion }}" -eq 8 ]; then
+            make phpunitforphp8.0 || (echo "First attempt failed. Retrying PHPUnit tests..." && make phpunitforphp8.0)
```

```suggestion
            make phpunitforphp8.0 || (echo "First attempt failed. Retrying PHPUnit tests..." && make phpunitforphp8.0)
```

If the command before this `||` fails, the command after this pipe `||` will be executed.